x/programs/examples/utils.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package examples
+
+import (
+	"context"
+	"os"
+
+	"github.com/ava-labs/avalanchego/database/memdb"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/crypto/ed25519"
+	"github.com/ava-labs/hypersdk/state"
+	"github.com/ava-labs/hypersdk/x/programs/runtime"
+	"github.com/near/borsh-go"
+)
+
+func newAddress() (ed25519.PrivateKey, codec.Address, error) {
```

Comment:
It was super annoying to make this generic in the hypersdk, recommend not assuming everything is ed25519 here.